components/runners/ambient-runner/ambient_runner/endpoints/tasks.py

```
@@ -0,0 +1,153 @@
+    if not re.match(r"^[a-zA-Z0-9_-]+$", task_id):
+        raise HTTPException(status_code=400, detail="Invalid task ID")
+
+    bridge = request.app.state.bridge
+    output_path = bridge.task_outputs.get(task_id)
+
+    # Fallback: search known directories where the SDK writes output files.
+    # During execution: /tmp/claude-*/.../{session_id}/tasks/{task_id}.output
+    # After completion: ~/.claude/projects/.../{session_id}/subagents/agent-{task_id}.jsonl
+    if not output_path:
+        search_dirs = [
+            Path("/tmp"),
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)